sql/src/main/java/io/crate/metadata/IngestionRuleInfos.java

```diff
@@ -0,0 +1,39 @@
+package io.crate.metadata;
+
+
+import io.crate.metadata.rule.ingest.IngestRulesMetaData;
+import org.elasticsearch.cluster.metadata.MetaData;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import static java.util.Collections.emptyIterator;
+
+public class IngestionRuleInfos implements Iterable<IngestionRuleInfo> {
+
+    private final MetaData metaData;
+
+    public IngestionRuleInfos(MetaData metaData) {
+        this.metaData = metaData;
```

Comment:
could be changed to require the IngestRulesMetaData directly or even the ingestRules directly - would make testing also easier